src/Umbraco.Core/Runtime/CoreRuntime.cs

```diff
@@ -147,8 +147,9 @@ protected virtual IFactory Boot(IRegister register, DisposableTimer timer)
 
                 // get composers, and compose
                 var composerTypes = ResolveComposerTypes(typeLoader);
+                var enableDisableAttributes = typeLoader.GetAssemblyAttributes(typeof(EnableComposerAttribute), typeof(DisableComposerAttribute));
                 composition.WithCollectionBuilder<ComponentCollectionBuilder>();
```

Comment:
Any reason this method call is placed between `composerTypes` and the initialization of `Composers`?

Reply:
@ronaldbarendse AFAIK calls to `WithCollectionBuilder` as empty calls (i.e. nothing proceeding it) adds this collection builder to the composition so that it can be resolved again with subsequent calls to `WithCollectionBuilder` in order to add to the collection. So potentially its so that this collection is added before the composers are created so the composers have access to this collection? I would assume this would need to be added before `composers.Compose()`

Reply:
Looks like it's not needed here, as the `composers.Compose();` method does the same:
https://github.com/umbraco/Umbraco-CMS/blob/932f7297f3473957c074325f72f632728eb8426a/src/Umbraco.Core/Composing/Composers.cs#L57-L60

Reply:
I just confirmed this gets called twice, so it's not required: the last commit removes the call within the `CoreRuntime`.